javers-core/src/test/java/org/javers/core/model/DummyUserPhone.java

```diff
@@ -0,0 +1,44 @@
+package org.javers.core.model;
+
+import org.javers.core.metamodel.annotation.ShallowReference;
+
+import javax.persistence.Id;
+
+/**
+ * @author akrystian
+ */
+@ShallowReference
+public class DummyUserPhone {
```

Comment:
A nie mógłbyś testować użycia tej klasy poprzez przykładowo dodanie pola w klasie: DummyUserDetails ? z tego co rozumiem to przez to, że jest oznaczona @ShallowReference to nic się nie powinno zmienic w innych testach